azure-pipelines.yml

```diff
@@ -18,225 +20,44 @@ stages:
-       env:
-         pass: $(dockerPassword)
-       displayName: Login to DockerHub
+    - name: 'env'
+      value: Development
 
-     - task: charleszipp.azure-pipelines-tasks-terraform.azure-pipelines-tasks-terraform-cli.TerraformCLI@0
-       displayName: 'Terraform Plan'
-       inputs:
-          command: plan
-          workingDirectory: $(Agent.BuildDirectory)/s/azure/terraform
-          environmentServiceName: 'azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef'
-          commandOptions: '-var="input_region=westeurope" -var="input_environment=Development" -var="input_container_version=$(Build.BuildNumber)"'
+    - name: 'KeyVaultId'
+      value : /subscriptions/8655985a-2f87-44d7-a541-0be9a8c2779d/resourceGroups/s118d01-secrets/providers/Microsoft.KeyVault/vaults/s118d01-secrets-kv
 
-     - task: charleszipp.azure-pipelines-tasks-terraform.azure-pipelines-tasks-terraform-cli.TerraformCLI@0
-       displayName: 'Terraform Apply'
-       inputs:
-         command: apply
-         workingDirectory: $(Agent.BuildDirectory)/s/azure/terraform
-         environmentServiceName: 'azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef'
-         commandOptions: '-var="input_region=westeurope" -var="input_environment=Development" -var="input_container_version=$(Build.BuildNumber)"'
+    - name: 'ResourceGroupName'
+      value: s118d01-app
 
-     - task: AzurePowerShell@5
-       displayName: 'Enable KV network firewall rule'
-       inputs:
-         azureSubscription: 'azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef'
-         ScriptType: InlineScript
-         Inline: |
-           # Setting Variables
-           $KeyVaultId = '/subscriptions/8655985a-2f87-44d7-a541-0be9a8c2779d/resourceGroups/s118d01-secrets/providers/Microsoft.KeyVault/vaults/s118d01-secrets-kv'
-           $KeyVault = Get-AzResource -ResourceId $KeyVaultId -ErrorAction SilentlyContinue
-           $KeyVault = Get-AzKeyVault -ResourceGroupName $KeyVault.ResourceGroupName -VaultName $KeyVault.Name
-           Write-Output 'Updating Key Vault rules...'
-           $KeyVault | Update-AzKeyVaultNetworkRuleSet -DefaultAction Deny
-         preferredAzurePowerShellVersion: 3.1.0
+    - name: 'SourceSlot'
+      value: staging
 
+    - name: 'WebAppName'
+      value: s118d01-app-as
 
-    # ########################SWAP SLOTS###########################################################
-     - task: AzureAppServiceManage@0
-       displayName: 'Swap Slots: s118d01-app-as'
-       inputs:
-         azureSubscription: 'azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef'
-         WebAppName: 's118d01-app-as'
-         ResourceGroupName: 's118d01-app'
-         SourceSlot: staging
-         timeoutInMinutes: 20
+    steps:
+    - template: templates/release.yml #Run Release Template
+      parameters :
+        ServiceConnection : azdo.pipelines.cip.S118D.armfe1ef140-8bef-4043-b5ee-c449e6f951ef
```

Comment:
Could this variable populated from variable group?. I would assume yes since its a string value